Tests/Marketplace/marketplace_services.py

```diff
@@ -1236,6 +1242,17 @@ def get_rn_files_names(self, modified_files_paths):
                 modified_rn_files.append(modified_file_path_parts[-1])
         return modified_rn_files
 
+    def only_release_notes_were_modified(self, modified_files_paths):
+        """
+        Check if all the paths in the modified files paths are of files in the ReleaseNotes directory.
+
+        Args:
+            modified_files_paths (list): list of paths of the pack's modified file
+        Returns:
+            bool: whether the files changed are only ReleaseNotes files
+        """
+        return all(self.RELEASE_NOTES in path for path in modified_files_paths)
+
```

Comment:
No need for a whole function for this, just use it as a one-line with a descriptive comment